builtin/mainmenu/tab_settings.lua

```diff
@@ -203,10 +200,15 @@ local function formspec(tabview, name, tabdata)
 
 	if core.settings:get("touchscreen_threshold") ~= nil then
 		tab_string = tab_string ..
-			"label[4.3,4.2;" .. fgettext("Touchthreshold: (px)") .. "]" ..
-			"dropdown[4.25,4.65;3.5;dd_touchthreshold;0,10,20,30,40,50;" ..
+			"label[4.25,3.5;" .. fgettext("Touchthreshold: (px)") .. "]" ..
```

Comment:
please rename this to "Touch threshold" too

Reply:
```suggestion
			"label[4.25,3.5;" .. fgettext("Touch threshold: (px)") .. "]" ..
```

Reply:
done